### PR TITLE
Use less jQuery, alipinejs or vanillajs instead

### PR DESCRIPTION
**Pull request by frjo**
- [x] Convert toggle-related.js to vanillajs.
- [x] On submission detail convert toggle-actions-panel.js to alpinejs.
- [x] Convert submission-text-cleanup.js to vanillajs.
- [x] Replace dropdown.js with alpinejs.
- [x] Convert all-submissions-table.js to vanillajs.
- [x] Convert application-form-links-new-window.js to vanillajs.
- [x] Made PAF and SOW views and download buttons to look the same.

### CODE REVIEW

File: hypha/apply/funds/templates/funds/applicationsubmission_admin_detail.html [hunk 1]
```diff
@@ -7,9 +7,11 @@
 {% endblock %}
 
 {% block mobile_actions %}
-    <a class="js-actions-toggle button button--white button--full-width button--actions">{% trans "Actions to take" %}</a>
-    <div class="js-actions-sidebar sidebar__inner sidebar__inner--light-blue sidebar__inner--actions sidebar__inner--mobile">
-        {% include "funds/includes/admin_primary_actions.html"  %}
+    <div x-data="{ show: false }">
+        <button x-on:click="show = ! show" class="button button--white button--full-width button--actions">{% trans "Actions to take" %}</button>
+        <div x-show="show" x-transition class="js-actions-sidebar sidebar__inner sidebar__inner--light-blue">
```
Comment (theskumar) [inline hypha/apply/funds/templates/funds/applicationsubmission_admin_detail.html:12]: Remove `js-actions-sidebar` class?

File: hypha/apply/funds/templates/funds/applicationsubmission_admin_detail.html [hunk 2]
```diff
@@ -87,6 +89,5 @@ <h5>{% trans "Reviews & assignees" %}</h5>
     {{ block.super }}
     <script src="{% static 'js/jquery.fancybox.min.js' %}"></script>
     <script src="{% static 'js/fancybox-global.js' %}"></script>
-    <script src="{% static 'js/toggle-actions-panel.js' %}"></script>
```
Comment (theskumar) [inline hypha/apply/funds/templates/funds/applicationsubmission_admin_detail.html:90]: I believe this file is not used anywhere and can be removed. https://github.com/HyphaApp/hypha/blob/f85bdf2b329906fa713dd68ab4bf11f0e507a8f5/hypha/static_src/javascript/toggle-actions-panel.js

File: hypha/apply/funds/templates/funds/applicationsubmission_admin_detail.html (hunk 1 above)
Comment (theskumar) [reply hypha/apply/funds/templates/funds/applicationsubmission_admin_detail.html:12]: Ignore it, it seems it's used in the tests.

Comment (theskumar) [reply hypha/apply/funds/templates/funds/applicationsubmission_admin_detail.html:12]: Generally, It's preferred to use dedicated attributes as selector for testing DOM elements. e.g. `data-test-id="actions-siderbar"`, but this change can wait and doesn't need to be part of this this PR.

**Comment by wes-otf** [comment]
all looked good on test from what I could see!

**Pull request merged**